src/opcodes/opcodes.cairo

```diff
@@ -264,4 +265,176 @@ pub mod Opcode {
             _ => utils::not_implemented(ref engine)
         }
     }
+
+    pub fn check_opcode(opcode: u8, ref engine: Engine) -> Result<(), felt252> {
```

Comment:
I think this implementation of this function will be a bit slow, and is hard to read and maintain.

I'd recommend using an if statement with `||` expressions instead for all the possible disabled opcodes. Also, calling the function something like `is_opcode_disabled` would be more accurate.

Reply:
Working on it!